src/include/storage/recovery/recovery_manager.h

```diff
@@ -159,19 +165,25 @@ class RecoveryManager : public common::DedicatedThreadOwner {
    * Recovers the databases using the provided log provider
    * @return number of committed transactions replayed
    */
-  void Recover() { RecoverFromLogs(); }
+  void Recover(bool catalog_only = false) { RecoverFromLogs(catalog_only); }
```

Comment:
Comments to explain function argument

Reply:
See comment from above. No default argument values in any function